lib/dbservice_web/views/resource_view.ex

```diff
@@ -30,6 +34,9 @@ defmodule DbserviceWeb.ResourceView do
       learning_objective_id: resource.learning_objective_id,
       tag_ids: resource.tag_ids,
       teacher_id: resource.teacher_id,
+      exam_ids: resource.exam_ids,
+      exam_details: exam_details,
+      show_in_gurukul: resource.show_in_gurukul,
```

Comment:
This shouldn't be on the DB as a resource param. Why does `db-service` need to know about Gurukul?